src/script/search/SearchService.ts

```diff
@@ -17,16 +17,17 @@
  *
  */
 
-import {getLogger} from 'Util/Logger';
+import {APIClient} from '@wireapp/api-client';
+import {SearchResult} from '@wireapp/api-client/dist/user';
 
 export class SearchService {
+  private readonly apiClient: APIClient;
   /**
    * Construct a new Search Service.
-   * @param {BackendClient} backendClient - Client for the API calls
+   * @param {APIClient} apiClient - Client for the API calls
```

Comment:
```suggestion
   * @param apiClient Client for the API calls
```

Reply:
Let's remove the whole comment block since the types are self-explaining.